core/src/main/scala/org/http4s/util/Renderable.scala

```diff
@@ -93,6 +96,11 @@ object Renderer {
       override def render(writer: Writer, values: Set[T]): writer.type =
         writer.addSet(values)
     }
+
+  implicit def headerSelectRenderer[A](implicit select: v2.Header.Select[A]): Renderer[A] =
```

Comment:
I think this is the only case where an http4s model type appears in this file.  If it works putting this on `Select` instead of `Renderer`, I think that would be preferrable.  I also just want to get this all compiling again, so I'm content to revisit this later.